Integrations/integration-Tanium.yml

```diff
@@ -72,13 +75,41 @@ script:
     except Exception:
         raise
 
-    def csvstr_to_list(str):
-        lines = str.splitlines()
+    def parse_xml_to_csv(instring):
+        output = StringIO()
+        root = ET.fromstring(instring)
+        columns = []
+        for column in root.findall('./result_set/cs/c/dn'):
+            columns.append(column.text)
+        rows =[]
+        rows.append(columns)
+        for record in root.findall('./result_set/rs/r'):
+            values = record.findall('c/v')
+            # todo bug with count > 2 use limit=1 for  now in index file query
```

Comment:
Why is this limited for count <= 1

Reply:
This is a comment and not coded.  Basically if you find multiple instances of the same hash on a single machine it will disrupt the script automation that is also part of this PR.  It is expecting one hash found per machine to output results.

Reply:
Can't we tackle this issue on our end? I'm not sure that's a good solution, @shaniacht1 what do you think

Reply:
There isn't a bug here.  The bug is in the automation script.  
Thus in the automation script i set limit=1.

Reply:
I can just remove this comment.